test/use-swr-streaming-ssr.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { Suspense } from 'react'
```

Comment:
I would like to add e2e test using playwright for this 

since i already finished the most of jobs in #2252 

I think we should nextjs app-dir here ?

Reply:
Sounds good! ❤️